11ty/definitions/-misia.md

```diff
@@ -0,0 +1,44 @@
+---
+title: -misia
+slug: -misia
+speech: noun
+defined: true
+sub_terms:
+  - text: Fat
+    full_title: fatmisia
+  - text: Islam
+    full_title: fatmisia
+  - text: Trans
+  full_title: transmisia
+reading:
+  - text: Beatley Library Anti-Oppression Guide
+    href: https://simmons.libguides.com/anti-oppression#s-lib-ctab-10174165-1
+  - text: 'Anti-Oppression: Anti-Fatmisia'
+    href: https://simmons.libguides.com/anti-oppression/anti-fatmisia
+---
+---
+from Greek for hate or hatred
+
+## Use
+
+-Misia can be appended to minoritised identifiers that are targeted for hate, such as to fat (fatmisia), trans (transmisia), or Islam (Islamomisia).
+
+## Issues
+
+-Phobia is Greek for 'fear of'.
+
+When used as a suffix in the context of Islamophobia, transphobia, or fatphobia, it is translated to mean that the group has a fear of individuals and communities who identify in those ways. However, the implication and subtext typically is that there is prejudice and discrimination against those groups. It falsely masks hate as fear.
+
+Additionally, phobias are one real component of anxiety disorders and mental illness, so conflating prejudice and discrimination (things that can be helped) with anxiety disorders (things that cannot necessarily be helped) additionally harms people who experience phobias from their anxiety disorders.
```

Comment:
I suggest replacing “one” with “a” in the first sentence (“Additionally, phobias are a real...”). And possibly remove real (although I see where you’re going with that, it reads a little oddly to me). 

For the two parenthetical should, maybe “attitudes and behaviours that can be changed” for the first and “medical conditions that cannot necessarily be changed” for the second? “Things” seems a bit vague and unclear to me. 

For the last bit, maybe “phobias as a result of their anxiety disorder”.